email_backend/data_management/generate_csv.js

```diff
@@ -0,0 +1,92 @@
+import { connect_db } from "../src/db_utils/connect_db";
+import { get_templates } from "../src/template_utils";
+import { make_mongoose_model_from_original_template } from "../src/db_utils/log_email_and_meta_to_db";
+import _ from "lodash";
+import fs from "fs";
+import mongoose from "mongoose";
+import { Parser } from "json2csv";
+
+const json2csv_parser = new Parser();
+
+connect_db()
+  .then(() => {
+    const templates = get_templates();
+    let templates_left = _.keys(templates).length;
+    _.forEach(templates, (template_value, template_name) => {
+      const collection = make_mongoose_model_from_original_template({
```

Comment:
Should note, this is a `Model` instance, not a mongoDB collection. The mongoose model is just, among other things, to interact with the associated DB collection.